Sources/CredentialsFacebook/TypeSafeFacebookToken.swift

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright IBM Corporation 2018
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+
+import Kitura
+import KituraNet
+import LoggerAPI
+import Credentials
+import Foundation
+import TypeDecoder
+
+/// A protocol that a user's type can conform to representing a user authenticated using a
```

Comment:
The description in lines 25-26 is not being displayed and the usage example is not being rendered as Swift code, it's just showing as text.